fetch.bs

```diff
@@ -1088,8 +1119,12 @@ must be an algorithm accepting an exception.
     </ol>
```

Comment:
Would've commented further up, but there's a call to incrementally-read loop (line 1113) that doesn't pass the timing info struct along, and probably should

Reply:
ack